packages/functional-tests/tests/syncV3/fxDesktopHandshake.spec.ts

```diff
@@ -2,24 +2,19 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { SYNC_EMAIL_PREFIX } from '../../lib/accountManager';
 import { FirefoxCommand, createCustomEventDetail } from '../../lib/channels';
-import {
-  expect,
-  test,
-  PASSWORD,
-  SIGNIN_EMAIL_PREFIX,
-  SYNC_EMAIL_PREFIX,
-} from '../../lib/fixtures/standard';
+import { expect, test } from '../../lib/fixtures/standard';
 import uaStrings from '../../lib/ua-strings';
 
 test.describe.configure({ mode: 'parallel' });
 
 test.describe('severity-2 #smoke', () => {
-  test.beforeEach(async () => {
-    test.slow();
-  });
-
   test.describe('Firefox desktop user info handshake', () => {
+    test.beforeEach(async () => {
+      test.slow();
```

Comment:
I'm starting to question the liberal use of `.slow()` vs just changing a global timeout value in a config [and accepting that some non-`.slow()` tests will have a more generous default timeout].

Reply:
I made a task for this a few weeks ago --> FXA-9388